github/resource_github_actions_organization_secret_test.go

```diff
@@ -0,0 +1,117 @@
+package github
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccGithubActionsOrganizationSecret(t *testing.T) {
+	t.Run("creates and updates secrets without error", func(t *testing.T) {
+		secretValue := "super_secret_value"
+		updatedSecretValue := "updated_super_secret_value"
+
+		config := fmt.Sprintf(`
+			resource "github_actions_organization_secret" "test_secret" {
+			  secret_name      = "test_secret_name"
+			  plaintext_value  = "%s"
+			}
```

Comment:
[Tests seems to be failing](https://github.com/terraform-providers/terraform-provider-github/actions/runs/361785493) due to a missing parameter here:

```
2020-11-13T14:56:31.9129101Z --- FAIL: TestAccGithubActionsOrganizationSecret (0.03s)
2020-11-13T14:56:31.9130511Z     --- FAIL: TestAccGithubActionsOrganizationSecret/creates_and_updates_secrets_without_error (0.02s)
2020-11-13T14:56:31.9132146Z         --- SKIP: TestAccGithubActionsOrganizationSecret/creates_and_updates_secrets_without_error/with_an_anonymous_account (0.00s)
2020-11-13T14:56:31.9133346Z             resource_github_actions_organization_secret_test.go:70: anonymous account not supported for this operation
2020-11-13T14:56:31.9134760Z         --- SKIP: TestAccGithubActionsOrganizationSecret/creates_and_updates_secrets_without_error/with_an_individual_account (0.00s)
2020-11-13T14:56:31.9135969Z             resource_github_actions_organization_secret_test.go:74: individual account not supported for this operation
2020-11-13T14:56:31.9137406Z         --- FAIL: TestAccGithubActionsOrganizationSecret/creates_and_updates_secrets_without_error/with_an_organization_account (0.02s)
2020-11-13T14:56:31.9138701Z             testing.go:654: Step 0 error: config is invalid: Missing required argument: The argument "visibility" is required, but no definition was found.
2020-11-13T14:56:31.9140034Z     --- FAIL: TestAccGithubActionsOrganizationSecret/deletes_secrets_without_error (0.01s)
2020-11-13T14:56:31.9142351Z         --- SKIP: TestAccGithubActionsOrganizationSecret/deletes_secrets_without_error/with_an_anonymous_account (0.00s)
2020-11-13T14:56:31.9143617Z             resource_github_actions_organization_secret_test.go:106: anonymous account not supported for this operation
2020-11-13T14:56:31.9145133Z         --- SKIP: TestAccGithubActionsOrganizationSecret/deletes_secrets_without_error/with_an_individual_account (0.00s)
2020-11-13T14:56:31.9146407Z             resource_github_actions_organization_secret_test.go:110: individual account not supported for this operation
2020-11-13T14:56:31.9147905Z         --- FAIL: TestAccGithubActionsOrganizationSecret/deletes_secrets_without_error/with_an_organization_account (0.01s)
2020-11-13T14:56:31.9149270Z             testing.go:654: Step 0 error: config is invalid: Missing required argument: The argument "visibility" is required, but no definition was found.
2020-11-13T14:56:31.9149955Z FAIL
2020-11-13T14:56:31.9150779Z FAIL	github.com/terraform-providers/terraform-provider-github/github	0.041s
2020-11-13T14:56:31.9151544Z FAIL
```